Cargo.toml

```diff
@@ -30,6 +30,7 @@ codecov = { repository = "rust-cli/clap-verbosity-flag" }
 [dependencies]
 log = "0.4.1"
```

Comment:
How do tracing users generally feel about still requiring `log`?  If a breaking change is involved in the future, should we allow disabling it?

Reply:
> How do tracing users generally feel about still requiring `log`?

I'm only a very new user of `tracing`, so I don't know, but I note that the main `tracing` README suggests that applications use the crate `tracing_subscriber`, which already, as a default feature, depends on `log` and consumes `log` messages, and the main `tracing` crate also interfaces with `log` as a non-default feature.

I'm sure some users would consider the cost in compilation time of `log` to be too much, but would they be using `clap-derive`?